platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## 5.0.2 - July 29, 2019
+* Fixed an issue where fill extrusion layers would be incorrectly rendered above other layers. ([#15065](https://github.com/mapbox/mapbox-gl-native/pull/15065))
+* Fixed a map update bug caused by the render tiles and the render passes becoming unsynchronized. ([#15092](https://github.com/mapbox/mapbox-gl-native/pull/15092))
+* Fixed a custom geometry source bug caused by using the outdated tiles after style update. ([#15112](https://github.com/mapbox/mapbox-gl-native/pull/15112))
```

Comment:
Use actual class names, because iOS developers aren’t going to know that “custom geometry source” means MGLComputedShapeSource (not CustomGeometrySource) and might not easily make the connection from “fill extrusion layer” to MGLFillExtrusionStyleLayer.

Reply:
Oh, sorry for piling on. This browser tab got stale. 😅